jekyll/_docs/faq.md

```diff
@@ -6,40 +6,61 @@ categories: [getting-started]
 description: Frequently Asked Questions
 ---
 
+## How do I subscribe to CircleCI announcements and get updates about new features and changes?
+
+We recommend that **all customers subscribe to the '[Announcements](https://discuss.circleci.com/c/announcements)' category** on our forums. You will get advance notice of important changes and new features.
+
+You can **subscribe by email, RSS or JSON**. **[Follow these instructions](https://discuss.circleci.com/t/how-to-subscribe-to-announcements-and-notifications-from-circleci-email-rss-json/5616) to subscribe now.**
+
+You may also like to follow our **[changelog](https://circleci.com/changelog/)** to see what has been released.
+
+Our **[status page](http://status.circleci.com/)** gives information on current system status, outages and past incidents.
+
 ## Authentication
 
 ### Do you support BitBucket or GitLab?
-Currently we support authentication with GitHub and Bitbucket. We do look at feature requests on [Discuss](https://discuss.circleci.com/) if you want to show support for other version control systems.
+
+Currently we support authentication with GitHub and Bitbucket. Please vote on relevant [Feature REquest](https://discuss.circleci.com/c/feature-request) if you want to show support for other version control systems.
 
 ### I can’t give CircleCI the access to all my private repositories. What do I do?
+
+Please see our guides and explanations about [GitHub and Privacy](https://circleci.com/docs/github-privacy/)
```

Comment:
We shouldn't link https://circleci.com/docs/github-privacy/ here as this guide/page is going away shortly (the whole category it's in actually is going away).